### ISSUE TITLE
Support using Personal Access Tokens instead of plain text password storing

### ISSUE THREAD

**Issue by G-Rath** [zenhob/hcl#97]
This should also solve #94

https://help.getharvest.com/api-v2/authentication-api/authentication/authentication/

**Comment by mkarnicki** [comment]
I don't think it will resolve #94 - I created a personal access token on an account within an org that has Google Login enabled - I can only access my user info via the API, all other endpoints return HTTP 403.

**Comment by fwolfst** [comment]
> I don't think it will resolve #94 - I created a personal access token on an account within an org that has Google Login enabled - I can only access my user info via the API, all other endpoints return HTTP 403.

Pretty sure I have another experience, but will have to evaluate. Plaintext password is a no-go here unfortunately, too.

**Comment by fwolfst** [comment]
Unfortunately personal (non-oauth) access tokens are not supported in v1 of the harvest API that is used by this gem.
